### PR TITLE
Removed twig deprecations

### PR DESCRIPTION
**Pull request by DonCallisto**
Twig deprecations removed (when you'll boost minimum version of sf to 3.2 you need twig 1.28 and there are some deprecations that we need to fix).

Se comments I'm gonna leave

### CODE REVIEW

File: Twig/DataGridExtension.php [hunk 1]
```diff
@@ -357,27 +364,27 @@ public function getPagerfanta($grid)
...
      *
      * @throws \InvalidArgumentException If the block could not be found
+     *
+     * @return string
      */
     protected function renderBlock(\Twig_Environment $environment, $name, $parameters)
     {
         foreach ($this->getTemplates($environment) as $template) {
-            if ($template->hasBlock($name)) {
+            if ($template->hasBlock($name, [])) {
```
Comment (DonCallisto) [inline Twig/DataGridExtension.php]: If second parameter is missing, this function is deprecated. As we weren't passing anything, I suppose that empty array is the right choice. Feel free to disagree.

File: Twig/DataGridExtension.php [hunk 2]
```diff
@@ -386,17 +393,18 @@ protected function renderBlock(\Twig_Environment $environment, $name, $parameter
...
      *
-     * @return boolean
+     * @return bool
      */
     protected function hasBlock(\Twig_Environment $environment, $name)
     {
         foreach ($this->getTemplates($environment) as $template) {
-            if ($template->hasBlock($name)) {
+            /** @var $template \Twig_Template */
+            if ($template->hasBlock($name, [])) {
```
Comment (DonCallisto) [inline Twig/DataGridExtension.php]: Same as above comment

**Comment by hmert** [comment]
@DonCallisto Could you resolve/update this PR too?

**Comment by DonCallisto** [comment]
@hmert done

**Pull request merged**